20211011.html

```diff
@@ -0,0 +1,16 @@
+<html>
+
+<head>
+  <link rel="stylesheet" href="./20211011.css">
+</head>
```

Comment:
`<meta charset="UTF-8">` 追加お願いします :pray:

```suggestion
<head>
  <meta charset="UTF-8">
  <link rel="stylesheet" href="./20211011.css">
</head>
```